p/php-src/LICENSE

```diff
@@ -0,0 +1,68 @@
+--------------------------------------------------------------------
```

Comment:
@vinodk99  we will need apache license in here.. right?
for build script license is always apache

Reply:
@nikhil-kalbande added apache-license